sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -149,10 +151,10 @@
 
   /**
    * Creates an uninitialized {@link BigtableIO.Read}. Before use, the {@code Read} must be
-   * initialized with a
-   * {@link BigtableIO.Read#withBigtableOptions(BigtableOptions) BigtableOptions} that specifies
-   * the source Cloud Bigtable instance, and a {@link BigtableIO.Read#withTableId tableId} that
-   * specifies which table to read. A {@link RowFilter} may also optionally be specified using
+   * initialized with a {@link BigtableIO.Read#withInstanceId(ValueProvider)} and
```

Comment:
The default usage will be the String values rather than ValueProviders for ease of use and brevity of code.  Can you please use `{@link BigtableIO.Read#withInstanceId(String)}` and etc in the comments?

Reply:
Ok.